sdk/python/tests/integration/registration/test_universal_cli.py

```diff
@@ -151,7 +157,9 @@ def test_nullable_online_store(test_nullable_online_store) -> None:
         try:
             repo_path = Path(repo_dir_name)
             feature_store_yaml = make_feature_store_yaml(
-                project, test_nullable_online_store, repo_path
+                project,
```

Comment:
you missed this one?

Reply:
This doesn't require the fixture since it uses nullable_online_store_configs which only tests aws, gcp, and file.